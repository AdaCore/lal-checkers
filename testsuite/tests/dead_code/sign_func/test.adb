function Sign(X : Integer)
   return Integer is
begin
   if X >= 1 then
      return 1;
   elsif X > 0 then
      return -1;
   else
      return 0;
   end if;
end Sign;