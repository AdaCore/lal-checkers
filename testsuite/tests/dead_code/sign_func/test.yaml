driver: python
helper: test_dead_code.py
test_subprogram: Sign