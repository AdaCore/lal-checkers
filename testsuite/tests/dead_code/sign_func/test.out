{
  "always": [
    "assign1",
    "assume2"
  ],
  "le_t_eq_v": [
    "assign1",
    "assume2"
  ]
}